sw/host/tests/crypto/rsa_kat/src/main.rs

```diff
@@ -87,7 +87,12 @@ fn run_rsa_testcase(
     };
 
     // Convert the inputs into the expected format for the CL.
-    let n: Vec<_> = test_case.n.iter().copied().rev().collect();
+    let mut n: Vec<_> = test_case.n.iter().copied().rev().collect();
+    // n in the wycheproof vectors seem to start with a leading 0.
+    if n.len() != test_case.security_level {
+        // Remove it.
+        n.pop();
+    }
```

Comment:
```suggestion
        n.pop();
        assert_eq!(n.pop(), Some(0));
    }
    assert_eq!(n.len(), test_case.security_level);
```
Nit: this is pedantic, but given the added vector manipulation I think it would be worth introducing a couple of additional assertions to explicitly error on any unexpected cases.

(if the previous comment is relevant, the final assert should instead be `assert_eq!(n.len() * u8::BITS as usize, test_case.security_level)`)